internal/spentcalories/spentCalories.go

```diff
@@ -15,6 +19,21 @@ const (
 
 func parseTraining(data string) (int, string, time.Duration, error) {
 	// ваш код ниже
+	parts := strings.Split(data, ",")
+	if len(parts) != 3 {
+		return 0, "", 0, errors.New("Ошибка")
```

Comment:
Нужно давать больше информации
```suggestion
		return 0, "", 0.0, fmt.Errorf("elements must be 3")
```

Reply:
Нужно так же везде переделать